src/frontend/with-page-auth-required.tsx

```diff
@@ -81,7 +82,7 @@ export type WithPageAuthRequired = <P extends WithPageAuthRequiredProps>(
  * @ignore
  */
 const withPageAuthRequired: WithPageAuthRequired = (Component, options = {}) => {
-  return function withPageAuthRequired(props): JSX.Element {
+  function WithPageAuthRequired(props: WithPageAuthRequiredProps): JSX.Element {
```

Comment:
I think the correct type would be `props: P` but I don't know how to forward the generic from the type `WithPageAuthRequired` without duplicating the types. 🤔